lib/taskana-cdi/src/main/java/pro/taskana/common/internal/TaskanaProducers.java

```diff
@@ -47,9 +47,9 @@ public void init() {
       ctx = new InitialContext();
       properties.load(propertyStream);
       dataSource = (DataSource) ctx.lookup(properties.getProperty("datasource.jndi"));
-      try (Connection connection = dataSource.getConnection()) {
-        DatabaseMetaData metaData = connection.getMetaData();
-        if (LOGGER.isDebugEnabled()) {
+      if (LOGGER.isDebugEnabled()) {
+        try (Connection connection = dataSource.getConnection()) {
```

Comment:
Does it make sense to get the connection while debugging only?

Reply:
Yes, at this time the connection is only needed for debugging

Reply:
ok